client/state/media/actions.js

```diff
@@ -225,6 +227,35 @@ export const deleteMediaItem = ( siteId, mediaId ) => ( {
 	mediaId,
 } );
 
+/**
+ * Returns an action object used in signalling that a media item was deleted or updated.
+ *
+ * @param {number} siteId site identifier
+ * @param {object} mediaItem media item
+ * @param {string} mediaAction action to pass on to Gutenframe ('updated' or 'deleted')
+ * @returns {object} action object
+ */
+export const addGutenframeMediaAction = ( siteId, mediaItem, mediaAction ) => ( {
+	type: ADD_GUTENFRAME_MEDIA_ACTION,
+	siteId,
+	mediaItem,
```

Comment:
Do we need to store the entire media item object? Would be great if we can use what we already have in the query manager; otherwise, we end up with a duplicated state for the same thing, which could also potentially end up being stale at some point.